test/cli/repl/t10-avalanche/expected

```diff
@@ -40,16 +40,25 @@ c$conv$5 =
     \conv$4 : (Int, DateTime). 
     add# [Int] c$conv$5 (1 : Int))
     (0 : Int) conv$2
-conv$6 =
-    rlatest [Int] (3 : Int)
+conv$9 =
+    rfold   [Int] [Buf Int] (
+    \conv$10 : Buf Int.  
+    \conv$11 : Int. 
+    Latest_push# [Int] conv$10
+    conv$11)
+    ([ExceptScalarVariableNotAvailable
+     ,ExceptScalarVariableNotAvailable
+     ,ExceptScalarVariableNotAvailable] : Buf Int)
```

Comment:
I think it's OK, but I think VBuf should change to (VBuf Int [BaseValue]) or whatever, so this would actually be (Buf 3 [])